bindings/bindingtester/bindingtester.py

```diff
@@ -47,14 +45,36 @@
 import fdb
 import fdb.tuple
 
+sys.path[:0] = [os.path.join(os.path.dirname(__file__), "..")]
```

Comment:
This should be before importing fdb, because it defines the path to find the fdb package. Binding tester is possible not going to work after this change.

In general I would avoid reformatting this file as part of the version bump. It is an old file and we should probably reformat it, but not necessarily as a part of this PR.

Reply:
Yep, that's why I'd made the [note](https://github.com/apple/foundationdb/pull/9572#issuecomment-1456802774) earlier about the reformat due to pre-commit. I'll remove the reformat from this PR with the workaround suggested by Josh.

Reply:
I just do git commit --no-verify, when I want to avoid the pre-commit formatting check